src/components/ManageUsers/UserSearch.jsx

```diff
@@ -0,0 +1,25 @@
+const UserSearch = () => {
+  return (
+    <main>
```

Comment:
main tag here is not the best option. We already have a container which is main. You can remove it and leave only `usersearch__container` div

Reply:
Main tag was removed as you suggested.